src/LaunchDarkly.XamarinSdk/ILdClient.cs

```diff
@@ -237,10 +237,7 @@ public interface ILdClient : IDisposable
         /// numeric metric value.
         /// </summary>
         /// <remarks>
-        /// As of this version’s release date, the LaunchDarkly service does not support the <c>metricValue</c>
-        /// parameter. As a result, calling this overload of <c>Track</c> will not yet produce any different
-        /// behavior from calling <see cref="Track(String, LdValue)"/> without a <c>metricValue</c>.
-        /// Refer to the <a href="https://docs.launchdarkly.com/docs/xamarin-sdk-reference#section-track">SDK reference guide</a>
+        /// Refer to the <a href="https://docs.launchdarkly.com/sdk/features/events#xamarin">SDK reference guide</a>
         /// for the latest status.
```

Comment:
Sorry, I should have included lines 240-241 in the obsolete section to be deleted. The "refer to ... for the latest status" part doesn't make sense now - it was talking about the "status" of the feature not having been implemented yet.

Reply:
```suggestion
```